### ISSUE TITLE
MD030 (list-marker-space) produces buggy warning message

### ISSUE THREAD

**Issue by EvgenyOrekhov** [DavidAnson/markdownlint#57]
```md
#

-

1.

```

The above code produces the following warnings:

```txt
3 - MD030 Spaces after list markers [Expected: 1; Actual: true]
5 - MD030 Spaces after list markers [Expected: 1; Actual: true]
```

Something is not right here.

The bug is reproducible on the [markdownlint demo](https://dlaa.me/markdownlint/) page.
